### PR TITLE
Add information about reviewing Dependabot PRs

### PR DESCRIPTION
**Pull request by hannalaakso**
Adding some information about how to review Dependabot PRs.

The rest of the doc could possibly do with a bit of restructuring now, but we could do that in a separate PR if we thought it was useful.

### CODE REVIEW

**Comment by EoinShaughnessy** [comment]
Is it worth linking this PR to [#1787](https://github.com/alphagov/govuk-design-system/issues/1787)?

File: development/code-reviews.md [hunk 1]
```diff
@@ -60,3 +60,11 @@ GitHub’s ‘update branch’ button does)
 
 Update the relevant Trello ticket for the PM to review the change (or should
 this happen before the merge?)
+
+## When you review a Dependabot PR
+
+You should check that the pull request:
```
Comment (EoinShaughnessy) [inline development/code-reviews.md:66]:
```suggestion
You should make sure the pull request:
```

File: development/code-reviews.md [hunk 2]
```diff
@@ -60,3 +60,11 @@ GitHub’s ‘update branch’ button does)
...
+ - works correctly
+ - does not introduce unexpected or undesired functionality
```
Comment (EoinShaughnessy) [inline development/code-reviews.md:68]:
```suggestion
 - does not introduce unexpected or unwanted functionality
```

File: development/code-reviews.md [hunk 3]
```diff
@@ -60,3 +60,11 @@ GitHub’s ‘update branch’ button does)
...
+
+After you have approved the pull request, ask another developer from the team to review it. After the other developer has approved the pull request, they merge it.
```
Comment (EoinShaughnessy) [inline development/code-reviews.md:70]:
```suggestion
To make sure the pull request works correctly:

1. Look at where the package is used with `npm ls`.
2. Test that the features using the package work correctly.

After you have approve the pull request, ask another developer from the team to review it. After the other developer approves the pull request, they can merge it.
```

**Comment by hannalaakso** [comment]
@EoinShaughnessy I've created a draft of the the things that we discussed in dev time last week to help us review Dependatbot PRs, would you mind having a look: https://docs.google.com/document/d/1lQ0QBLxmpGUNYOl_RQKTxJRcLUJ8AQPe6Etq4_su0yo/edit#heading=h.v1pvpnf6ajfb

**Comment by EoinShaughnessy** [comment]
@hannalaakso Content looks just about oven-ready! Left a couple of small comments.

**Comment by vanitabarrett** [comment]
@hannalaakso @EoinShaughnessy Thanks both, the doc looks good to me 👍

File: development/code-reviews.md [hunk 4]
```diff
@@ -60,3 +60,13 @@ GitHub’s ‘update branch’ button does)
...
+
+## When you review a Dependabot PR
+
+To make sure the pull request works correctly:
+
+- Read the release notes for the new version and look for changes that would impact our codebase. This is especially important with major releases.
+- Look at where in our codebase the package is used with `npm ls` - [see this example, where tests, Rollup and linting used the updated package](https://github.com/alphagov/govuk-design-system/pull/1813#issuecomment-896654240). Test that the parts of the codebase using the package work correctly.
+- If the new version is a major release, consider waiting for a few days before updating to see if anyone reports issues that could impact us.
+
+After you approve the pull request, ask another developer from the team to review it. After the other developer approves the pull request, they can merge it.
```
Comment (vanitabarrett) [inline development/code-reviews.md:72]: Is it worth making this more generic - i.e: the person reading this documentation may already be the second developer reviewing the dependabot PR, so they wouldn't go and ask someone else. Maybe it's enough to say that dependabot PRs should be treated like external contributions and require 2 approving reviews before merging?

Comment (hannalaakso) [reply development/code-reviews.md]: Updated, thanks to @EoinShaughnessy for word wrangling help.

**Review by vanitabarrett — APPROVED** [review]

**Pull request merged**